addmanager.go

```diff
@@ -1,19 +1,41 @@
 package main
 
-import "fmt"
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+)
+
+// addProj takes in a project name and adds it to the workspace.
+// Requires that the workspace exists.
+// Requires that the project name is non-empty
+// * tested
+func addProj(projName string, wsPath string) {
+	ws, err := ioutil.ReadFile(wsPath)
+	check(err)
+	err = os.MkdirAll(string(ws)+"/"+projName, os.ModePerm)
+	check(err)
+	fmt.Printf("Added project %s to workspace %s\n", projName, string(ws))
+}
 
 // ? Incomplete functionality
 // addProjectHandler takes in a project name and initializes a new project.
 // If the project name is empty, it prompts the user to enter a name.
 // Vanilla behavior includes generating a directory labeled the project name.
 // Initializes all boilerplate code for specified project type.
-// ! untested
+// * tested
 func addProjectHandler(projName string) error {
```

Comment:
what's the point of making this an error return type if it always returns nil?

Reply:
Yea, that's a good point. I think technically the function can return nothing, the way I wrote them.